tools/pot/docs/SaturationIssue.md

```diff
@@ -1,21 +1,21 @@
+The 8-bit instructions of previous generations of Intel CPUs (based on SSE, AVX-2, AVX-512 instruction sets) admit so-called saturation (overflow) of the intermediate buffer when calculating the dot product which is an essential part of Convolutional or MatMul operations. This saturation can lead to an accuracy drop on the mentioned architectures during the inference of 8-bit quantized models. However, such degradation is not possible to predict, since most of the computations are executed in parallel during DL model inference which makes this process non-deterministic. This is a common problem for models with non-ReLU activation functions and low level of redundancy (for example, optimized or efficient models). It can prevent deploying the model on legacy hardware or creating cross-platform applications. The problem does not occur on the CPUs with Intel Deep Learning Boost (VNNI) technology and further generations, as well as on GPUs.
 
 ## Saturation Problem Detection
 The only way to detect saturation issue is to run inference on the CPU that admits it and on the hardware that does not have such problem (for example, VNNI-based CPU). If the accuracy difference is significant (more than 1%), this is the main indicator of the saturation issue impact.
 
 ## Workaround
-There is a workaround that helps fully address the saturation issue during the inference. The algorithm uses only 7 bits to represent weights (of Convolutional or Fully-Connected layers) while quantizing activations using the full range of 8-bit data types. However, this can lead to an accuracy degradation due to the reduced representation of weights. On the other hand, using this workaround for the first layer can help mitigate the saturation issue for many models.
+The algorithm uses only 7 bits to represent weights (of Convolutional or Fully-Connected layers), while quantizing activations using the full range of 8-bit data types. Using this workaround for the first layer can help mitigate the saturation issue for many models. However, this can lead to an accuracy degradation due to the reduced representation of weights.
```

Comment:
```suggestion
The algorithm uses only 7 bits to represent weights (of Convolutional or Fully-Connected layers), while quantizing activations using the full range of 8-bit data types. Using this workaround for the first layer can help mitigate the saturation issue for many models. However, this can lead to the degradation of accuracy due to the reduced representation of weights.
```